scripts/sv/manage_sv_pipeline.sh

```diff
@@ -0,0 +1,174 @@
+if [ "${GATK_GIT_HASH}" != "${CURRENT_GIT_HASH}" ]; then
+        while true; do
+                read -p "Current git hash does not match GATK git hash. Run anyway?" yn
+                case $yn in
+                        [Yy]*)  break
+                                ;;
+                        [Nn]*)  exit
+                                ;;
+                        *)      echo "Please answer yes or no"
+                        
+                esac
+        done
+fi
+GIT_BRANCH=$(git -C ${GATK_DIR} branch --contains ${GATK_GIT_HASH} | rev | cut -d" " -f 1 | rev)
+# set output directory to datetime-git branch-git hash stamped folder
+OUTPUT_DIR="/results/$(date "+%Y-%m-%d_%H.%M.%S")-${GIT_BRANCH}-${GATK_GIT_HASH}"
+
+# call create_cluster, using default_init
+while true; do
+    echo "#############################################################" 2>&1 | tee -a ${LOCAL_LOG_FILE}
+    read -p "Create cluster? (yes/no/cancel)" yn
+    case $yn in
+        [Yy]*)  if [[ ${INIT_SCRIPT} == gs://* ]]; then
+                        INIT_ARGS=${INIT_SCRIPT}
+                else
+                        INIT_ARGS="${INIT_SCRIPT} gs://${PROJECT_NAME}/${GCS_USER}/init/$(basename INIT_SCRIPT)"
+                fi
+
+                echo "create_cluster.sh ${GATK_DIR} ${PROJECT_NAME} ${CLUSTER_NAME} ${GCS_REFERENCE_DIR} ${GCS_BAM_DIR} ${INIT_ARGS} 2>&1 | tee -a ${LOCAL_LOG_FILE}" | tee -a ${LOCAL_LOG_FILE}
+                create_cluster.sh ${GATK_DIR} ${PROJECT_NAME} ${CLUSTER_NAME} ${GCS_REFERENCE_DIR} ${GCS_BAM_DIR} ${INIT_ARGS} 2>&1 | tee -a ${LOCAL_LOG_FILE}
+                break
+                ;;
+        [Nn]*)  break
+                ;;
+        [Cc]*)  exit
+                ;;
+        *)      echo "Please answer yes, no, or cancel."
+                ;;
+    esac
+done
+
+# call runWholePipeline
+while true; do
+    echo "#############################################################" 2>&1 | tee -a ${LOCAL_LOG_FILE}
+    read -p "Run whole pipeline? (yes/no/cancel)" yn
+    case $yn in
+        [Yy]*)  SECONDS=0
+                echo "runWholePipeline.sh ${GATK_DIR} ${CLUSTER_NAME} ${OUTPUT_DIR} ${GCS_BAM} ${GCS_REFERENCE_FASTA} ${GCS_REFERENCE_IMAGE}${SV_ARGS} 2>&1 | tee -a ${LOCAL_LOG_FILE}" | tee -a ${LOCAL_LOG_FILE}
+                runWholePipeline.sh ${GATK_DIR} ${CLUSTER_NAME} ${OUTPUT_DIR} ${GCS_BAM} ${GCS_REFERENCE_FASTA} ${GCS_REFERENCE_IMAGE}${SV_ARGS} 2>&1 | tee -a ${LOCAL_LOG_FILE}
+                printf 'Pipeline completed in %02dh:%02dm:%02ds\n' $((${SECONDS}/3600)) $((${SECONDS}%3600/60)) $((${SECONDS}%60))
+                break
+                ;;
+        [Nn]*)  break
+                ;;
+        [Cc]*)  exit
+                ;;
+        *)      echo "Please answer yes, no, or cancel."
+                ;;
+    esac
+done
+
+# copy results into gcloud
+while true; do
+    echo "#############################################################" 2>&1 | tee -a ${LOCAL_LOG_FILE}
+    read -p "Copy results? (yes/no/cancel)" yn
```

Comment:
It depends. They may have run previously (perhaps not using this script, or perhaps there was an error). If there's data, the copy script should copy it off, and regardless, it should copy any log files or command line arguments.